llvm/test/Transforms/DeadArgElim/sycl-kernels-neg1.ll

```diff
@@ -0,0 +1,13 @@
+; TODO: this should be crashing (not --crash) after enabling assert
```

Comment:
We can just do not transform kernel functions. This seems to be more user friendly than crash.
What do you think?

Reply:
That's a good point. We can even skip scheduling the pass if the option is not set.

Reply:
Introduced an early exit if `IntegrationHeaderFileName` is not provided.